mods/tuxemon/l18n/fr_FR/LC_MESSAGES/base.po

```diff
@@ -6240,34 +6240,37 @@ msgstr ""
 
 ## COMBAT STATE STATUSES
 msgid "combat_state_blinded_get"
-msgstr "{target} est aveuglé."
+msgstr "{target} ne voit plus rien."
 
 msgid "combat_state_chargedup_get"
 msgstr "{target} est complètement chargé."
 
 msgid "combat_state_charging_get"
-msgstr "{target} augmente la puissance."
+msgstr "{target} augmente son attaque."
 
 msgid "combat_state_confused_get"
 msgstr "{target} est confus."
 
 msgid "combat_state_diehard_get"
-msgstr "{target} continuera à se battre."
+msgstr "{target} se battra jusqu'au bout."
+
+msgid "combat_state_diehard_tech"
+msgstr "{target} se bat malgré la douleur."
 
 msgid "combat_state_dozing_get"
 msgstr "{target} est endormi."
 
 msgid "combat_state_enraged_get"
-msgstr "{target} est furieux."
+msgstr "{target} bouillit de rage."
```

Comment:
Not sure, but shouldn't we use present tense in this case?
So "bout de rage."
https://dictionnaire.lerobert.com/definition/bouillir